my-app/src/components/Body/Content/Content.jsx

```diff
@@ -1,9 +1,20 @@
 import React from "react";
 import BlogView from "./BlogView/BlogView";
+import CreateView from "./CreateView/CreateView";
+import {
+    Switch,
+    Route,
+} from "react-router-dom";
 
 const Content = () => {
-    return(
-        <BlogView/>
+    return (
+        <Switch>
+            <Route exact path="/" render={() => <BlogView/>}>
+            </Route>
+            <Route exact path="/createView" render={() => <CreateView/>}>
```

Comment:
use this format
`<Route exact path="/createView" component={CreateView} />`

this format 
`<Route exact path="/createView" render={() => <CreateView/>}>`
used only if you need to drill some props